vendor/VisualAcceptance.js

```diff
@@ -76,24 +74,23 @@ function capture(imageName, width, height, misMatchPercentageMargin, assert) {
       })
       $(document.getElementById('ember-testing')).removeAttr('style')
       $(document.getElementById('ember-testing-container')).removeAttr('style')
-      node.innerHTML = '<div class="test pass"> <div class="list-name"> No passed image. Saving current as baseline: ' + imageName + '</div> <div class="additional-info"> Addition Information: </div> <img src="'+ image + '" /> </div>'
+      node.innerHTML = '<div class="test pass"> <div class="list-name"> No passed image. Saving current as baseline: ' + imageName + '</div> <div class="additional-info"> Addition Information: </div> <img src="' + image + '" /> </div>'
```

Comment:
Will get fixed with this https://github.com/ciena-blueplanet/ember-cli-visual-acceptance/pull/44